src/main/java/svc/data/citations/CitationDAO.java

```diff
@@ -1,17 +1,26 @@
 package svc.data.citations;
 
 import com.google.common.collect.Lists;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.PreparedStatementCallback;
 import org.springframework.stereotype.Component;
 import rx.Observable;
+import svc.data.jdbc.BaseJdbcDao;
+import svc.logging.LogSystem;
 import svc.models.Citation;
+import svc.util.DatabaseUtilities;
 
 import javax.inject.Inject;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
 import java.time.LocalDate;
+import java.util.ArrayList;
 import java.util.List;
 
 //NOTE: If we switch to groovy, we can greatly reduce code here since we can pass the function as an argument to a method
 @Component
-public class CitationDAO {
+public class CitationDAO extends BaseJdbcDao {
```

Comment:
All the code you've merged into here no longer belongs. CitationDAO no longer houses mock/test data. You should put your code into the MockCitationDataSource